local/compose.go

```diff
@@ -98,7 +99,12 @@ func (s *composeService) Up(ctx context.Context, project *types.Project, detach
 		}
 	}
 
-	err = inDependencyOrder(ctx, project, func(c context.Context, service types.ServiceConfig) error {
+	err = s.ensureImagesExists(ctx, project)
```

Comment:
Nit: this is already at the start of the method ; probably a rebase issue, remove this `ensureImagesExists()`

Reply:
will fix this and rebase anyway...